# This message is used for debugging purposes. It helps extract the path planning data to be plotted in matlab
#     Each set of coefficients are the coefficients of a 3rd degree polynomial representing the desired path

Header header
std_msgs/Bool isIntercept
geometry_msgs/Vector3 intercept_pose
geometry_msgs/Vector3 intercept_point
float64 intercept_radius
float64 intercept_time
